src/models/CommonModel.ts

```diff
@@ -20,6 +20,102 @@ export class CommonModel extends CommonSchema<CommonModel> {
   extend?: string[];
   originalSchema?: Schema | boolean;
   
+  /**
+   * Retrieves data from originalSchema by given key
+   * 
+   * @param key given key
+   * @returns {any}
+   */
+  getFromSchema<K extends keyof Schema>(key: K) {
+    let schema = this.originalSchema || {};
+    if (typeof schema === 'boolean') schema = {};
+    return schema[`${key}`];
+  }
+
+  /**
+   * Set the types of the model
+   * 
+   * @param types to set the model type to
+   */
+  setType(types : string | string[]) {
+    if (Array.isArray(types)) {
+      if (types.length === 0) {
```

Comment:
You may want to unnest if`s like this one with something similar to `if(!Array.isArray(types) { this.types = types; return; }`

Reply:
Refactored it 😄